app/views/applications/personal_statement.html

```diff
@@ -17,8 +17,18 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
 
 
       {% set guidanceHtml %}
-        <p class="govuk-body">You should write between 500 and 1000 words for your personal statement.</p>
-        <p class="govuk-body">Here are some examples you could write about (they are just a guide, you do not need to write about all of them):</p>
+
+        {% if lastApplication %}
+          <div class="govuk-inset-text">
+           <p class="govuk-body">We've copied your personal statement from your previous application.</p>
+
+           <p class="govuk-body">You can make changes to your statement for this application, or leave it the same.</p>
+          </div>
+        {% endif %}
+
+
+        <p class="govuk-body">You should write between 500 words for your personal statement.</p>
+        <p class="govuk-body">Here are some examples you could write about:</p>
```

Comment:
I think this lead-in sentence needs changing as it doesn't lead into the bullets (they'd be a full stop after it with the way its worded at the moment). How about, 'You could include details about:' or 'You could include information such as:'.